.github/workflows/commits.yml

```diff
@@ -2,15 +2,38 @@
 name: Conventional commit check
 on: [push]
 jobs:
+<<<<<<< HEAD
```

Comment:
The conflict resolution didn't work

Reply:
Strange that the CI didn't fail.
Anyway, I'm ready to approve once it is fixed.

Reply:
It failed but it is not showing on the PR https://github.com/cartesi/rollups/actions/runs/5082931365

Reply:
Ok, It was an IDE error that messed me up. Fixed.